### PR TITLE
recipe for rubicon-ml, a python package

### PR DESCRIPTION
**Pull request by joe-wolfe21**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ryanSoley** [comment]
I am willing to be listed as a maintainer

**Comment by mmccarty** [comment]
Yes, I'm willing to be listed as a maintainer.

**Comment by gforsyth** [comment]
Hey @conda-forge/help-python this is ready for review!

**Comment by dianelee217** [comment]
I am willing to be a maintainer

File: recipes/rubicon-ml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - rubicon=rubicon.cli:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - click >=7.1
+    - dask[dataframe] >=2.12.0
```
Comment (mariusvniekerk) [inline recipes/rubicon-ml/meta.yaml:26]:
```suggestion
    - dask >=2.12.0
    - pandas >=1.0
```

you probably want this

Comment (joe-wolfe21) [reply recipes/rubicon-ml/meta.yaml]: thanks, updated

**Comment by gforsyth** [comment]
Thanks for catching the bad dependency @mariusvniekerk -- this is ready for another look.

**Pull request merged**